xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEStream.cpp

```diff
@@ -276,10 +276,12 @@ unsigned int CActiveAEStream::AddData(uint8_t* const *data, unsigned int offset,
           if (m_format.m_streamInfo.m_type == CAEStreamInfo::STREAM_TYPE_TRUEHD)
           {
             m_currentBuffer->pkt->nb_samples += 2560;
-            uint8_t highByte = (minFrames >> 8) & 0xFF;
```

Comment:
i want to keep this statement on a separate line. multiple statements on a single line are hell when it comes to debugging.